packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -438,8 +438,12 @@ class CloudFormation {
     const cfnModel = this.cfn;
 
     return new Promise((resolve, reject) => {
-      cfnModel.describeStacks(cfnStackParams, err => {
+      cfnModel.describeStacks(cfnStackParams, (err, data) => {
         const cfnDeleteStatus = 'stackDeleteComplete';
+        if ((err && err.message.includes(`${stackName} does not exist`)) || data.StackStatus === 'DELETE_COMPLETE') {
```

Comment:
The code is a bad request 400, I thought it would send a 404 but that was not case. The DELETE_COMPLETE is sent if it was recently deleted

Reply:
If thats the case

> The code is a bad request 400, I thought it would send a 404 but that was not case. The DELETE_COMPLETE is sent if it was recently deleted

If thats the case, lets first check the error code and and then check for message.